### ISSUE TITLE
Displaying images broken - nextcloud 3.29.3

### ISSUE THREAD

**Issue by Mis1ify** [nextcloud/android#13515]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. Update Nextcloud APP from 3.29.0 to 3.29.3
2. Open random image in app
3. Instead of a big preview, you see small thumbnail, square, and an information on the bottom asking should I download full image?

### Expected behaviour

Images should open correctly after you click on it, in big preview, like on version 3.29.0 of the app. Downgrading from 3.29.3 to 3.29.0 fixed the issue.

### Actual behaviour

Images open in a small preview after you click on it, in a small thumbnail.

### Android version

14

### Device brand and model

Xiaomi POCO F3 (Alioth)

### Stock or custom OS?

Custom (explain in "additional information")

### Nextcloud android app version

3.29.3

### Nextcloud server version

29.0.5.1

### Using a reverse proxy?

Yes

### Server error logs

```bash
{"reqId":"Ztdh265S1-Fi9_gY7y6s1gAAAA8","level":3,"time":"2024-09-03T19:22:03+00:00","remoteAddr":"192.168.1.100","user":"--","app":"no app in context","method":"GET","url":"/index.php/core/preview?fileId=31159&x=256&y=256&a=1&mode=cover&forceIcon=0","message":"Could not decrypt or decode encrypted session data","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.29.3","version":"29.0.5.1","exception":{"Exception":"Exception","Message":"HMAC does not match.","Code":0,"Trace":[{"file":"/var/www/nextcloud/lib/private/Security/Crypto.php","line":119,"function":"decryptWithoutSecret","class":"OC\\Security\\Crypto","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/Session/CryptoSessionData.php","line":91,"function":"decrypt","class":"OC\\Security\\Crypto","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/Session/CryptoSessionData.php","line":68,"function":"initializeSession","class":"OC\\Session\\CryptoSessionData","type":"->"},{"file":"/var/www/nextcloud/lib/private/Session/CryptoWrapper.php","line":112,"function":"__construct","class":"OC\\Session\\CryptoSessionData","type":"->"},{"file":"/var/www/nextcloud/lib/base.php","line":458,"function":"wrapSession","class":"OC\\Session\\CryptoWrapper","type":"->"},{"file":"/var/www/nextcloud/lib/base.php","line":714,"function":"initSession","class":"OC","type":"::"},{"file":"/var/www/nextcloud/lib/base.php","line":1184,"function":"init","class":"OC","type":"::"},{"file":"/var/www/nextcloud/index.php","line":47,"args":["/var/www/nextcloud/lib/base.php"],"function":"require_once"}],"File":"/var/www/nextcloud/lib/private/Security/Crypto.php","Line":158,"message":"Could not decrypt or decode encrypted session data","exception":[],"CustomMessage":"Could not decrypt or decode encrypted session data"},"id":"66d88035aceed"}
```

### Additional information

Using LineageOS 21, app downloaded straight from github, non-gplay release. Sorry for no android logs, I was annoyed and downgraded immediately. 
For the server logs, these are the only one that match the exact time, however they could be not related, because IP given here matches my Home Assistant server, not my phone.

**Comment by himpierre** [comment]
Same here. 3.30 affected too.

**Comment by Mis1ify** [comment]
> Same here. 3.30 affected too.

Thx for telling this, I just won't update the app until it'll be fixed.

**Comment by himpierre** [comment]
Still broken with 3.30.1, right? Cheers!

Am 14. September 2024 12:56:31 MESZ schrieb Mis1ify ***@***.***>:
>> Same here. 3.30 affected too.
>
>Thx for telling this, I just won't update the app until it'll be fixed.
>
>
>

**Comment by Mis1ify** [comment]
It's pretty important feature... I and many other people use Nextcloud mainly for storing images, and this bug totally breaks the experience :/ I hope they'll fix it, everything was right until version 3.29.3 came out. In version 3.29.2 everything was working fine!
